### PR TITLE
add rating to bike list

### PR DESCRIPTION
**Pull request by miladmikeal**

### CODE REVIEW

**Comment by anderja8** [comment]
I think it looks a little too crowded with all of the current information on the list. My vote is to remove the `style` column.
image

**Comment by miladmikeal** [comment]
I'm fine with removing style.

File: src/components/BrowseBikesListItem.jsx [hunk 1]
```diff
@@ -1,81 +1,123 @@
...
+            <TouchableOpacity
+              onPress={() => {
+                setSelectedBikeID(bike.id);
+              }}
+            >
+              <Text>Frame</Text>
+              <Text note>{bike.frame}</Text>
+            </TouchableOpacity>
+          </Col>
+          <Col>
```
Comment (anderja8) [inline src/components/BrowseBikesListItem.jsx:57]: I say we just remove this Col component.

File: src/components/BrowseBikesListItem.jsx [hunk 2]
```diff
@@ -1,81 +1,123 @@
...
+              <Text note>{bike.style}</Text>
+            </TouchableOpacity>
+          </Col>
+          <Col>
+            <TouchableOpacity
+              onPress={() => {
+                setSelectedBikeID(bike.id);
+              }}
+            >
+              <Text>Dist</Text>
```
Comment (anderja8) [inline src/components/BrowseBikesListItem.jsx:73]: We can get away with changing this back to distance now that the style column is gone.

**Review by anderja8 — APPROVED** [review]

**Pull request merged**